ctapipe/calib/camera/gainselection.py

```diff
@@ -51,12 +50,12 @@ def pick_gain_channel(waveforms, threshold, select_by_sample=False):
             new_waveforms[gain_mask] = waveforms[1][gain_mask]
 
     elif waveforms.shape[0] == 1:
-        new_waveforms = np.squeeze(waveforms)
+        new_waveforms = np.squeeze(waveforms, axis=0)
```

Comment:
Why use np.squeeze instead of waveforms[0]?

Reply:
not sure - that was just copied from Tino's code.. I will change it to `waveforms[0]` since it makes more sense

Reply:
Just out of curiosity, I see multiple mentions of something called 'Tinos Code'. Is this something one should know about?...

Reply:
>Just out of curiosity, I see multiple mentions of something called 'Tinos Code'. Is this something one should know about?...

Sorry that may be confusing: Tino (@tino-michael) is the one who created the full pipeline based on ctapipe that has been used so far to make sensitivity curves. To get it working, he had to create a few extra classes and scripts that are in his github repo, and they are not yet merged into ctapipe (this PR is the first step to doing that, but there will be several others once this is working).  It turned out to be not as simple as just copying his code in, since we wanted to move things around a bit, like having the gain selection in R1, rather than in Dl1 as it is in his code, hence some refactoring.

Reply:
I think I used squeeze and not just the index because at some point there was an inconsistency somewhere with the time dimension. sometimes the whole dimension was gone, sometimes it was there but with length 1...
as in the different behaviour of `a = [1, 2, 3]` and `b = [[1, 2, 3]]`

Reply:
I think that problem with missing time dimensions, etc, has been fixed now in the new EventSource classes and the calibration code... at least I haven't seen it anymore with any of the test data I used.